### PR TITLE
VS-45752 Update dependencies for js-sdk

### PR DESCRIPTION
**Pull request by sandip-patel-1**
Motivation
------------

Design choices
------------

Side-effects/other
------------

Tests
------------
- all passing

Known issues/limitations
------------
- webpack was not able to upgrade to version 5

Release dependencies
------------

Who should review it
------------

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -2,59 +2,59 @@ var base64 = require('base-64');
...
-   * @return {String}
-   */
-  mergeNextPayload: function (args, nextPayload) {
-    if (Object.keys(nextPayload).length === 0) return args
-    // Merge potential query string params manually
-    if (nextPayload.params && args.params) {
-      var nextParams = nextPayload.params
-      for (var param in nextParams) {
-        if (typeof args.params[param] !== 'undefined') {
-          args.params[param] += (';' + nextParams[param])
```
Comment (bradconnolly) [inline src/utils.js:43]: Is this a safe change? Removing the `()`

File: webpack.config.js [hunk 2]
```diff
@@ -3,18 +3,19 @@
...
-        new webpack.IgnorePlugin(/vertx/),
-        new webpack.ProvidePlugin({
-          Promise: 'es6-promise-promise'
-        })
-    ]
+	mode: 'development',
+	entry: './src/timekit.js',
+	devtool: 'source-map',
+	output: {
+		path: __dirname + '/dist',
```
Comment (bradconnolly) [inline webpack.config.js:10]: Any reason we changed this?

File: webpack.config.min.js [hunk 3]
```diff
@@ -3,21 +3,22 @@
...
-        new webpack.optimize.DedupePlugin(),
-        new webpack.ProvidePlugin({
-          Promise: 'es6-promise-promise'
-        })
-    ]
+	mode: 'production',
+	entry: './src/timekit.js',
+	devtool: 'source-map',
+	output: {
+		path: __dirname + '/dist',
```
Comment (bradconnolly) [inline webpack.config.min.js:10]: This too

File: src/utils.js (hunk 1 above)
Comment (sandip-patel-1) [reply src/utils.js:43]: This change is a part of webpack upgrade from 1 to 4

File: webpack.config.min.js (hunk 3 above)
Comment (sandip-patel-1) [reply webpack.config.min.js:10]: This change is a part of webpack upgrade from 1 to 4

File: webpack.config.js (hunk 2 above)
Comment (sandip-patel-1) [reply webpack.config.js:10]: This change is a part of webpack upgrade from 1 to 4

File: src/utils.js (hunk 1 above)
Comment (sandip-patel-1) [reply src/utils.js:43]: this is much safer then without bracket

**Pull request merged**